src/browser/of_events.ts

```diff
@@ -42,6 +42,15 @@ class OFEvents extends EventEmitter {
 
         return super.emit(route, ...data);
     }
+
+    public SUBSCRIBER: StringMap = {
```

Comment:
yea trying not to edit 14 files just to introduce `OfEvents` which (due to o vs O) looks so much like `ofEvents`...

Regarding casing, I'll change it to option 3.